sdk/communication/Azure.Communication.Common/tests/CommunicationTestEnvironment.cs

```diff
@@ -28,11 +30,21 @@ public class CommunicationTestEnvironment : TestEnvironment
 
         public string CommunicationTestPhoneNumber => GetVariable(AzurePhoneNumber);
 
+        public string SkipSmsTest => GetOptionalVariable(SkipIntSmsTestEnvironmentVariableName) ?? "False";
+
+        public string SkipPhoneNumbersTest => GetOptionalVariable(SkipIntPhoneNumbersTestEnvironmentVariableName) ?? "False";
+
         public virtual string ExpectedTestPackagesEnabled { get { return TestPackagesEnabledDefaultValue; } }
 
         public bool ShouldIgnoreTests => TestPackagesEnabled != TestPackagesEnabledDefaultValue
             && TestPackagesEnabled != ExpectedTestPackagesEnabled;
 
+        public bool ShouldIgnoreSMSTests => TestPackagesEnabled != TestPackagesEnabledDefaultValue
+            && TestPackagesEnabled != ExpectedTestPackagesEnabled
+            || bool.Parse(SkipSmsTest);
+
+        public bool ShouldIgnorePhoneNumbersTests => bool.Parse(SkipPhoneNumbersTest);
```

Comment:
any reason why `ShouldIgnorePhoneNumbersTests` does not follow the same logic as `ShouldIgnoreSMSTests`?